docs/package.json

```diff
@@ -26,12 +26,15 @@
     "babel-plugin-transform-react-remove-prop-types": "^0.1.0",
     "babel-plugin-transform-runtime": "^6.3.13",
     "css-loader": "^0.23.0",
+    "doctrine": "^1.1.0",
     "eslint": "^1.10.3",
     "eslint-plugin-react": "^3.13.0",
+    "esutils": "^1.1.6",
```

Comment:
It's webpack's fault. esutils had updated their API with v2.0.2 yet doctrine uses the older 1.1.6 version. In a normal day everything works fine since semver is respected. but after digging around for 2 hours I found out that webpack was not respecting semver for this particular package. it would only bundle the 2.0.2 version and discard 1.1.6 hence. What doctrine was requiring was actually the 2.0.2 version not the 1.1.6 it was expecting! Doing this somehow forces webpack to contain both. I don't understand why webpack is doing something so stupid here. But this solves it.